### DISCUSSION TITLE
Mintaállományok

### DISCUSSION THREAD

**Discussion by tothj74** [nav-gov-hu/eVAT#52] [Általános]
A legutóbbi eÁFA workshopon ígért mintaállományok publikálása mikorra várható?
Nagyon megkönnyítené a fejlesztők munkáját, ha ezt mielőbb megkapnánk.

**Comment by renced42** [comment]
Kedves @tothj74 

2 db mintaállomány publikálásra került itt: https://github.com/nav-gov-hu/eVAT/tree/master/eVAT-M2M/sample%20xml
